packages/build-tools/src/utils/project.ts

```diff
@@ -19,6 +19,16 @@ export async function setup<TJob extends Job>(ctx: BuildContext<TJob>): Promise<
     }
   });
 
+  await ctx.runBuildPhase(BuildPhase.READ_PACKAGE_JSON, async () => {
+    try {
+      const packageJsonContents = await readPackageJson(ctx.reactNativeProjectDirectory);
+      ctx.logger.info('Using package.json:');
+      ctx.logger.info(JSON.stringify(packageJsonContents, null, 2));
+    } catch (err: any) {
+      ctx.logger.warn(`Failed to parse or read package.json: ${err.message}`);
+    }
```

Comment:
This is the existing function that I moved over from hooks. Didn't want to change it.

Reply:
So what would you like the output of the build phase to be if it fails 'silently'?

Reply:
I was suggesting to don't print anything so this would become:

```
    try {
      const packageJsonContents = await readPackageJson(ctx.reactNativeProjectDirectory);
      ctx.logger.info('Using package.json:');
      ctx.logger.info(JSON.stringify(packageJsonContents, null, 2));
    } catch (err: any) {
      // don't do anything
    }
```

However, if it's implemented the same in another place then I guess it's fine (... or we should update both places - I'll leave the decision for you).

Reply:
I think it's nice to see why nothing is logged if nothing is logged, so I'll stick with the existing implementation from hooks.